web/modules/custom/joinup_sparql/src/Database/Driver/sparql/Connection.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Drupal\joinup_sparql\Database\Driver\sparql;
+
+use Drupal\rdf_entity\Database\Driver\sparql\Connection as BaseConnection;
+use Drupal\rdf_entity\Database\Driver\sparql\ConnectionInterface;
+use Drupal\rdf_entity\Exception\SparqlQueryException;
+use EasyRdf\Http\Exception as EasyRdfException;
+use EasyRdf\Sparql\Result;
+
+/**
+ * @addtogroup database
+ * @{
+ */
+class Connection extends BaseConnection implements ConnectionInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function query(string $query): Result {
```

Comment:
>  Are you ok with that (counting the 5 seconds in the performance)?

I think that reflects the truth. But I don't have a strong opinion. The solution I proposed is not against adding the
```
if (!empty($this->logger)) {
  $this->logger->clear('webprofiler');
}
```
block of code. I think it could be added just before `sleep()`, if you think. I'll let you decide :)